draft-rfcs/working-group-terminology.md

```diff
@@ -2,96 +2,274 @@
 
-### Community Groups
+### Working Group Creation
 
-Community Groups are groups of individuals who want to work and collaborate on a specific topic, without going through the formal process of a working group. Community groups are free to adopt the structure of working or projects groups or create their own structure.
+Creation of a working group is approved by the core team. Typically this has
+been done by the core team agreeing to approve the creation of new working
+groups and having a period of time soliciting applications from the community,
+and then approving a subset of those applications.
 
-- Community groups are not required to follow any of the guidelines mentioned above.
-- As such community groups are not officially endorsed by the Rust project.
+> **DRAFT NOTE** Should this application come in the form of an RFC? My
+> inclination is yes, however it could just create more churn and drama than
+> needed. Posting in a thread on internals as was done previously might
+> be enough.
 
-# Drawbacks
+#### Application Checklist
+
+This not meant to be formal list of questions to be answer, however the
+application should try to address these points.
+
+- What value do you see your group bringing to the organisation?
+- Why should the project your group?
+- What are the goals of your group?
+- What are explicitly non-goals of your group?
+
+These first four points should also serve as the basis for the charter for the
+working group.
 
-[drawbacks]: #drawbacks
+- Is your group long-running or temporary?
+  - If it is temporary, how long do you see it running for?
+- What is the long-term vision of your group?
+- What do you expect the relationship to the organisation be?
+- How do you want to establish accountability?
+- If applicable, which other working groups or teams do you expect to have close
+  contact with?
+- What are the short-term goals of your group?
+- Who are the initial shepherds/leaders? (This is preferably 2–3 individuals,
+  but not required.)
+- How do you intend to make your work accessible to people outside your group?
+- Where do you see your group needing help?
+- What is your preferred way for the core team to contact you?
+
+### Working Group Evaluation
+
+Working group shepherds should be regularly participating in the project leads
+discussion channels so that the rest of the organisation is aware of the work
+and can help where needed.
+
+If a working group has demonstrated consistent productivity over a significant
+period time, and there is consensus that there is significant future work, it
+may become a Rust team. Conversely if there is consensus that the work is
+"complete" to the point that there's there is little benefit to continuing the
+working group. It may be wound down.
+
+The wind down process of a working group involves communicating the wind down to
+the community and the archival or transfer of ownership of the relevant projects.
+
+## Project Groups
```

Comment:
Do we want to formalize the notion that Project groups are intended to be the "canonical" way for "pre-RFC" collaboration to occur?

Reply:
Well I think we need more time with it. We can say this is some teams intend to have pre-RFC, but I would want to have a couple of retrospectives and see if there are issues when others try to adopt it before we give a "canonical" endorsement.